### DISCUSSION TITLE
Practica Pastelería

### DISCUSSION THREAD

**Discussion by JesusGarridoM** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#580] [Show and tell]
Buenos días compañeros, les quiero mostrar mi pagina, no es tan compleja porque aun sigo aprendiendo pero es trabajo honesto 😁 

> Link pagina: [https://agreeable-sky-046b3440f.1.azurestaticapps.net](https://agreeable-sky-046b3440f.1.azurestaticapps.net)
> 
> Link de repo: [https://github.com/JesusGarridoM/FRONTEND](https://github.com/JesusGarridoM/FRONTEND)

Se aceptan sugerencias y comentarios 😉

**Comment by DanyVeneno** [comment]
Hola Jesus en tu link pagina despliega un 404

**Reply by JesusGarridoM** [reply]
Gracias, ya lo corregí

**Comment by DanyVeneno** [comment]
Esta chida bien funcional

**Comment by JuanSalvador73** [comment]
Muy buen sitio, felicidades.

**Comment by RodolfoMorquecho** [comment]
Te quedo muy bien, por cierto sabes si existe algún formulario para la entrega de esa practica?

**Reply by JesusGarridoM** [reply]
Gracias y con respecto a tu pregunta, para esta práctica ya no subieron formulario 🙂
